from_cpython/Objects/complexobject.c

```diff
@@ -356,15 +354,20 @@ PyComplex_AsCComplex(PyObject *op)
     }
 }
 
+// Pyston changes: comment this out
+#if 0
 static void
 complex_dealloc(PyObject *op)
 {
     op->ob_type->tp_free(op);
 }
+#endif
 
 
-static PyObject *
-complex_format(PyComplexObject *v, int precision, char format_code)
+// Pyston change: make no static
```

Comment:
I'm wondering why you had to change the signature?
I think every call site has a BoxedComplex* so it could just pass that one in instead of creating a ```Py_complex``` just to pass it around. (But we should add static asserts which make sure that BoxedComplex::imag/real have the same offsets as CAPI PyComplexObject::cval.imag/real)